Modules/_testmultiphase.c

```diff
@@ -952,3 +952,37 @@ PyInit__test_shared_gil_only(void)
 {
     return PyModuleDef_Init(&shared_gil_only_def);
 }
+
+
+#include "datetime.h"
+
+static int
+datetime_capi_client_exec(PyObject *m)
+{
+    _PyDateTimeAPI_Clear();
+    if (PyDateTimeAPI != NULL) {
+        return -1;
+    }
+    PyDateTime_IMPORT;
```

Comment:
I would prefer to call _PyDateTimeAPI_Import() directly, or better: have a function which reports errors.